### PR TITLE
Translation for PT-BR

### PR DESCRIPTION
**Pull request by fclesio**
@VictorGarritano or @haneybarg can you guys perform a review in this one? I didn't translate some terms that we do not have in PT-BR or terms that mean substantives (_e.g._ Reformer, Transformer, BERT, etc).

I've included some italics for foreign terms according to ABNT for facilitating the reading.

### CODE REVIEW

File: _posts/2020-01-19-NLP_Newsletter[PT-BR]_Reformer,_DeepMath,_ELECTRA,_TinyB.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+title: "NLP Newsletter: Reformer, DeepMath, ELECTRA, TinyBERT para busca, VizSeq, Open-Sourcing ML,…"
+author: billy_rick
```
Comment (omarsar) [inline _posts/2020-01-19-NLP_Newsletter[PT-BR]_Reformer,_DeepMath,_ELECTRA,_TinyB.md]: Hi @fclesio you may want the change the author name to the one you provided: fclesio

**Comment by omarsar** [comment]
@fclesio For terms that you cannot find a translation for, it's okay to leave them as they are. Thanks.

**Comment by fclesio** [comment]
Can you merge?

No worries. Thanks a lot!

**Comment by omarsar** [comment]
@fclesio I left something for review. If you would like your authorship info to appear in the post. You may want to change the `author` field to your username: `fclesio`

Let me know your choice and then I will merge the PR. Thank you! :)

File: _posts/2020-01-19-NLP_Newsletter[PT-BR]_Reformer,_DeepMath,_ELECTRA,_TinyB.md (hunk 1 above)
Comment (fclesio) [reply _posts/2020-01-19-NLP_Newsletter[PT-BR]_Reformer,_DeepMath,_ELECTRA,_TinyB.md]: I've included in the credits.

**Comment by fclesio** [comment]
If it's not possible to include 2 authors, just let your name and merge. 

Thanks once again.

**Review by omarsar — APPROVED** [review]
I updated it to just include your authorship information. It's all fine with me. :) Thanks again for doing this. I appreciate the effort. :)

**Pull request merged**